packages/stems/tsconfig.json

```diff
@@ -24,6 +24,6 @@
     "noUnusedParameters": true,
     "skipLibCheck": true
   },
-  "include": ["src"],
```

Comment:
whats the deal with this?

Reply:
not sure... will bring back. the directory gone soon anyway 🗡️